cmd/demo/validator.go

```diff
@@ -13,13 +13,14 @@ import (
 	"net"
 	"strconv"
 
+	"github.com/ethereum/go-ethereum/params"
 	"github.com/pkg/errors"
 	"perun.network/go-perun/log"
 	"perun.network/go-perun/wallet"
 )
 
 func valBal(input string) error {
-	_, err := strconv.ParseInt(input, 10, 32)
+	_, _, err := big.ParseFloat(input, 10, 64, big.ToNearestEven)
 	return errors.WithMessage(err, "Invalid Float")
```

Comment:
Please boyscout
```suggestion
	return errors.Warp(err, "parsing float")
```
The error didn't have a stack yet.

Reply:
That was actually me :see_no_evil:

Reply:
I know, that's why I wrote "boyscout" ;)